conf/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cd $(dirname $0)
+case "$1" in
+    web)
+        echo "Starting Web Server"
+        exec ./web.sh
+        ;;
+    worker)
+        echo "Starting Celery Worker"
+        exec ./worker.sh
+        ;;
+    test)
+        echo "Running Tests"
+        cd ..
+        flake8 . --exclude=./recommendation/views/data/dummy.py && nosetests
```

Comment:
it fails linting pretty badly. See the [circle build log #28](https://circleci.com/gh/mozilla/universal-search-recommendation/28) ... 

```
./recommendation/views/data/dummy.py:10:80: E501 line too long (92 > 79 characters)
./recommendation/views/data/dummy.py:16:80: E501 line too long (98 > 79 characters)
./recommendation/views/data/dummy.py:28:80: E501 line too long (114 > 79 characters)
./recommendation/views/data/dummy.py:34:80: E501 line too long (139 > 79 characters)
./recommendation/views/data/dummy.py:46:80: E501 line too long (89 > 79 characters)
./recommendation/views/data/dummy.py:52:80: E501 line too long (127 > 79 characters)
./recommendation/views/data/dummy.py:70:80: E501 line too long (106 > 79 characters)
./recommendation/views/data/dummy.py:82:80: E501 line too long (105 > 79 characters)
....
```

I'll leave it to you to decide the best way to make this file pass linting. Most of the violations appear to be an a large object definition and long URI values.